src/cpp/session/modules/SessionRenv.R

```diff
@@ -32,6 +38,20 @@
    
 })
 
+.rs.addFunction("renv.options", function()
+{
+   context <- .rs.renv.context()
+   
+   options <- list()
+   
+   options[["useRenv"]] <- context[["active"]]
+   # TODO: surface options associated with
```

Comment:
Nit: TODO should have your name or a github issue number (so we can always follow up with any dangling TODOs)